### PR TITLE
Added templated control item template for F#

### PR DESCRIPTION
**Pull request by Mrxx99**
This adds and item template for an TemplatedControl in F#

### CODE REVIEW

File: templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace AvaloniaAppTemplate.Namespace
+
+open Avalonia
+open Avalonia.Controls
```
Comment (Takoooooo) [inline templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs:4]: We don't need these two usings here,if it is copy-pasta from C# feel free to remove that from C# version also

**Comment by Takoooooo** [comment]
The only problem i see currently with our item templates is that they are not displayed in VS when you click Project -> Add ->New Item.
image
Thtas for F# and the same for C#. Do you know,can we somehow support that or it isn't supported by VS yet?
If it is not supported i will add our item templates to VS extension

File: templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs (hunk 1 above)
Comment (Mrxx99) [reply templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs:4]: Yes, that was copy pasta, will remove them from both

**Comment by Mrxx99** [comment]
> The only problem i see currently with our item templates is that they are not displayed in VS when you click Project -> Add ->New Item.
> image
> Thtas for F# and the same for C#. Do you know,can we somehow support that or it isn't supported by VS yet?
> If it is not supported i will add our item templates to VS extension

Yes I noticed that as well, I think that is a limitation from VS it seems to ignore CLI item templates. I uninstall the VS Avalonia extension and I had none item templates shown for Avalonia

File: templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs (hunk 1 above)
Comment (Mrxx99) [reply templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs]: done

**Comment by Mrxx99** [comment]
@Takoooooo You could make an issue for item template support here I think: https://github.com/dotnet/templating
Since it needs a new convention for a namespace parameter. Otherwise the experience would be really bad.
Maybe it would help moving that forward if the issue would be opened from the Avalonia account directly...

**Pull request merged**